.goreleaser.yaml

```diff
@@ -21,130 +21,24 @@ builds:
       - "-s -w -X pkg.Version={{.Version}} -X pkg.CommitID={{.Commit}} -buildid=''"
 
 dockers:
```

Comment:
Looks good! I think we are missing `build_flag_templates`:   

```
build_flag_templates:
  - "--platform=linux/arm64/v8"
```


(ref https://goreleaser.com/customization/docker_manifest/)

Reply:
i saw this one..but i got rid of it since im using `goos` instead

Reply:
The official goreleaser docs use `build_flag_templates:` for manifests instead of `goos` which is just set to linux so I'd prefer that, unless you have some reason to prefer `goos` (is it even equivalent for docker manifest builds?)

Reply:
my assumption was incorrect reading through https://github.com/goreleaser/goreleaser/blob/8e1b6a1bbb39712d92ea1b4bee2b424a59823522/www/docs/customization/docker.md?plain=1#L46

Fixed the platform